src/js/components/DataSort/stories/Simple.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Data, DataTable, Grid } from 'grommet';
+
+import { DataSort } from '../DataSort';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Simple = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Grid pad="large" columns={['large']} justifyContent="center">
+    <Data data={DATA} updateOn="change">
+      <DataSort />
```

Comment:
The sort by field needs a label (see accessibility panel on storybook)

Reply:
I checked with ANDI and only saw things that were a part of Select and RadioButtonGroup already.